app/controllers/explore.js

```diff
@@ -67,4 +67,21 @@ export default class ExploreController extends Controller {
       this.set('cfs', null);
     }
   }
+
+  @action
+  clearAllFilters() {
+    this.set('event_name', null);
```

Comment:
use instead ```this.setProperties({})```

Reply:
> use instead `this.setProperties({})`

any specific reason for using this.setProperties({})

Reply:
sometimes when ```this.set``` is called multiple times result in producing minor bugs